courses/models.py

```diff
@@ -205,7 +205,9 @@ def next_run_date(self):
             (
                 course_run.start_date
                 for course_run in self.courseruns.all()
-                if course_run.live and course_run.start_date > now
+                if course_run.live
+                and course_run.start_date
+                and course_run.start_date > now
```

Comment:
Is there any corresponding test e.g. `not start_data` ?

Reply:
Yes see https://github.com/mitodl/mitxpro/pull/836/files#diff-01d0d79c873a1164b798247647c5b421R232. Previously test also has an issue because `live=True` wasn't there and we were not having error due to `None` in `start_date`.